deeds-dapp-webapp/src/main/webapp/vue-app/dapp/commonRenting/components/button/RentButtons.vue

```diff
@@ -298,7 +299,7 @@ export default {
           event.stopPropagation();
         }
         this.$store.commit('setStandaloneOfferId', this.offers[0].id);
-        this.$root.$emit('switch-page', 'marketplace', true);
```

Comment:
Why, this name is technical in fact and shouldn't change, it's the ref name of Vue component.